taxi/tests/test_models.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Car, Manufacturer, Driver
+
+
+class ModelTests(TestCase):
+    def test_car_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="Test Manufacturer",
+            country="Test Country"
+        )
+        car = Car.objects.create(model="test", manufacturer=manufacturer)
+        self.assertEqual(str(car), car.model)
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="pididi",
+            first_name="Pididi",
+            last_name="P"
+        )
+        self.assertEqual(
+            str(driver),
+            f"{driver.username} ({driver.first_name} {driver.last_name})"
```

Comment:
Ensure that the `__str__` method of the `Driver` model is implemented to return the format `{driver.username} ({driver.first_name} {driver.last_name})`. If this is not the case, the test might fail. Verify the implementation in the `Driver` model.